lib/builtin.pm

```diff
@@ -0,0 +1,93 @@
+package builtin;
+
+use strict;
+use warnings;
+
+# All code, including &import, is implemented by always-present functions in
+# the perl interpreter itself.
+# See also `builtin.c` in perl source
+
+1;
+__END__
+
+=head1 NAME
+
+builtin - Perl pragma to import built-in utility functions
+
+=head1 SYNOPSIS
+
+    use builtin qw( true false isbool );
+
+=head1 DESCRIPTION
+
+Perl provides several utility functions in a package called C<builtin::>.
+Functions in this namespace can be called by their fully-qualified names, but
+it is often desired to give them shorter name aliases for convenience.
```

Comment:
This seems like a weird way to say they can be imported, as it suggests aliasing to a different name which isn't implemented. Perhaps can be combined with the following paragraph